### ISSUE TITLE
Sprint Not Working

### ISSUE THREAD

**Issue by gr8pefish** [breakinblocks/nutsandboltstorqued#192]
Using unmodified version of 1.6.4.7, and can't run.

Tried modifying keybinds, but it didn't seem to work. Will investigate further and will report back when I know more.

**Comment by Doomgull** [comment]
Using sponge?

**Comment by gr8pefish** [comment]
Nope, a normal MC server. Server downloaded from the CF page in fact.

**Comment by Doomgull** [comment]
Your instance of the pack is fucked, then, if nothing has been added/removed. Delete the whole folder, re-download, try again.

**Comment by gr8pefish** [comment]
Will do. Strange thing is that I think the sprint is working, you just don't go any faster: https://streamable.com/gcmrd

**Comment by SystemCollapse** [comment]
I can back this up, Sprint has not worked for me since day one.
